sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerClientJavaDocCodeSnippets.java

```diff
@@ -443,62 +443,57 @@ public void beginRecognizeReceiptsWithOptions() throws IOException {
-                    if (FieldValueType.DATE.equals(transactionDateField.getValueType())) {
-                        LocalDate transactionDate = FieldValueType.DATE.cast(transactionDateField);
-                        System.out.printf("Transaction Date: %s, confidence: %.2f%n",
-                            transactionDate, transactionDateField.getConfidence());
-                    }
+            }
+            FormField<?> transactionDateField = recognizedFields.get("TransactionDate");
+            if (transactionDateField != null) {
+                if (FieldValueType.DATE == transactionDateField.getValueType()) {
+                    LocalDate transactionDate = FieldValueType.DATE.cast(transactionDateField);
+                    System.out.printf("Transaction Date: %s, confidence: %.2f%n",
+                        transactionDate, transactionDateField.getConfidence());
                 }
-
-                FormField<?> receiptItemsField = recognizedFields.get("Items");
-                if (receiptItemsField != null) {
-                    System.out.printf("Receipt Items: %n");
-                    if (FieldValueType.LIST.equals(receiptItemsField.getValueType())) {
-                        List<FormField<?>> receiptItems = FieldValueType.LIST.cast(receiptItemsField);
-                        receiptItems.forEach(receiptItem -> {
-                            if (FieldValueType.MAP.equals(receiptItem.getValueType())) {
-                                Map<String, FormField<?>> formFieldMap = FieldValueType.MAP.cast(receiptItem);
-                                formFieldMap.forEach((key, formField) -> {
-                                    if ("Quantity".equals(key)) {
-                                        if (FieldValueType.DOUBLE.equals(formField.getValueType())) {
-                                            Float quantity = FieldValueType.DOUBLE.cast(formField);
-                                            System.out.printf("Quantity: %d, confidence: %.2f%n",
-                                                quantity, formField.getConfidence());
-                                        }
-                                    }
-                                });
+            }
+            FormField<?> receiptItemsField = recognizedFields.get("Items");
+            if (receiptItemsField != null) {
+                System.out.printf("Receipt Items: %n");
+                if (FieldValueType.LIST == receiptItemsField.getValueType()) {
+                    List<FormField<?>> receiptItems = FieldValueType.LIST.cast(receiptItemsField);
+                    receiptItems.stream()
+                        .filter(receiptItem -> FieldValueType.MAP == receiptItem.getValueType())
+                        .<Map<String, FormField<?>>>map(FieldValueType.MAP::cast)
+                        .forEach(formFieldMap -> formFieldMap.forEach((key, formField) -> {
+                            if ("Quantity".equals(key)) {
```

Comment:
is possible to use stream again and filter here?

Reply:
I guess I can. Just wanted to keep a method showing the non-stream way but thought that would be inconsistent. So kept all the sample files using `forEach`.
I don't have a strong preference, what do you think, update all of them?

Reply:
Personally I like to using stream() if possible.